gamedata/descriptions/levels/switch.md

```diff
@@ -0,0 +1,4 @@
+Just have to flip the switch. Can't be that hard, right?
+
+##### Things that might help:
+Understanding how `CALLDATA` is encoded.
```

Comment:
We've been talking at the forum to add reading material.  This [zeppelin article](https://blog.zeppelin.solutions/deconstructing-a-solidity-contract-part-iii-the-function-selector-6a9b6886ea49
) could be a good starting point for less knowledgeable players.

Reply:
Funny enough, that's also from @ajsantander, lord father of Ethernaut

Reply:
Those series are insanely great